x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/LocalStateMonitoring.java

```diff
@@ -89,4 +100,44 @@ public Monitoring getMonitoring() {
     protected Class<? extends TransportAction<XPackUsageRequest, XPackUsageResponse>> getUsageAction() {
         return MonitoringTransportXPackUsageAction.class;
     }
+
+    @Override
+    public List<ActionHandler<? extends ActionRequest, ? extends ActionResponse>> getActions() {
+        var actions = super.getActions();
```

Comment:
Additional note: It is not possible to add the ccr / enrich plugins (like with data streams), because the monitoring gradle project can't rely on these gradle modules, because both ccr and enrich gradle projects already depend on the monitoring gradle project.